RadarSDK/Radar.m

```diff
@@ -58,6 +66,11 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
     [RadarSettings setPublishableKey:publishableKey];
 
     RadarSdkConfiguration *sdkConfiguration = [RadarSettings sdkConfiguration];
+
+    if (NSClassFromString(@"XCTestCase") == nil) {
+        [Radar nativeSetup];
+    }
```

Comment:
This is before our GET config call, but it'd be nice if there was a way of turning this off remotely. Maybe we should check a feature flag regardless.

Reply:
Having this capability will be ideal, but this can result in unsafe behavior.
As of now, our API calls are not blocking and relies on callbacks. Calling `nativeSetup` within a callback is unsafe as there is no guarantee when this code will be called. 
Having the swizzle called later in the life cycle may result in missed logging of app opens via notifications.
If we are still interested in using background processing, registering background processes later in the lifecycle will crash the app.
We may need to consider configurations that we can pass in as variables to `initialize`.